pkg/dt/distributed_transaction_manger.go

```diff
@@ -238,11 +243,18 @@ func (manager *DistributedTransactionManager) processGlobalSessions() error {
 			if err != nil {
 				return err
 			}
+			// branch session has been committed or rollbacked
 			if len(bsKeys) == 0 {
 				if err := manager.storageDriver.DeleteGlobalSession(context.Background(), gs.XID); err != nil {
 					return err
 				}
 				log.Debugf("global session finished, key: %s", gs.XID)
+				switch gs.Status {
+				case api.Committing:
+					manager.recordGlobalTransactionMetric(gs.TransactionName, metrics.TransactionStatusCommitted)
```

Comment:
if a transaction is timeouted, how do we count the metric?

Reply:
good question